qabelbox/src/main/java/de/qabel/qabelbox/storage/model/BoxQuota.java

```diff
@@ -21,4 +21,14 @@ public void setQuota(long quota) {
         this.quota = quota;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (o instanceof BoxQuota) {
+            BoxQuota other = (BoxQuota) o;
+            if (other.quota == this.quota && other.size == this.size) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
guard clause
```JAVA
public boolean equals(Object o) {
    if (!(o instanceof BoxQuota) {
        return false;
    }
    BoxQuota other = (BoxQuota)o;
    return other.quota == quota && other.size == size);
}
```

plain kotlin translation
```KOTLIN
fun equals(other: Object) {
    if (other !is Boxquota) {
        return false
    }
    return other.quota == quota && other.size == size
}
```

maximum shortness
```KOTLIN
fun equals(Object other) = other is BoxQuota && other.quota == qutota && other.size == size
```

I don't insist on a change of the structure but please remove the unneccessary `this` for the sake of collective code ownership.